indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -407,10 +403,42 @@ $i-box-padding: 10px;
 
     > li {
         @include border-top();
-        display: block;
-        padding: 10px 10px;
+        padding: 10px;
         position: relative;
         overflow: hidden;
+
+        /* Sortable row with handle on the left */
```

Comment:
The only problem is the license header, that it uses `/* */`. Maybe we could update the script to maintain the headers to use `//` in `.scss` files and enable `no-css-comments` in `sass-lint`.